index.js

```diff
@@ -90,8 +93,49 @@ function handleClose (writeStream) {
       writeStream.emit('close')
     })
   }
+  function trapWindowsEPERMRename (err) {
+    if (err.syscall && err.syscall === 'rename' &&
+        err.code && err.code === 'EPERM' &&
+        writeStream.__isWin
+    ) {
+      if (fs.existsSync(writeStream.__atomicTarget)) {
```

Comment:
It strikes me as weird that this is a sync call and the `readFile` calls are async. Probably these should all be async.